app/pages/home-common/featured-project.jsx

```diff
@@ -9,20 +9,20 @@ Featured Project Component
 */
 
 import React from 'react';
-import {Link} from 'react-router'
+import { Link } from 'react-router';
 
-export default class FeaturedProject extends React.Component {
-  render() {
-    return (
-      <section className="home-featured">
-        <h1>Featured Project</h1>
-        <div className="home-featured-images">
-          <img src="./assets/featured-projects/featured-project-20170407-etchacell.jpg" />
-        </div>
-        <h2>New Project Launch - Etch a Cell</h2>
-        <p>The Zooniverse has just launched a brand new project to help improve understanding of cell biology.</p>
-        <Link to="/projects/h-spiers/etch-a-cell" className="alternate-button">Get involved!</Link>
-      </section>
-    );
-  }
-}
+const FeaturedProject = () => {
```

Comment:
Declare this as a function, rather than assigning an anonymous function to a variable. `function FeaturedProject() {…}`